matter_server/common/helpers/json.py

```diff
@@ -22,8 +21,6 @@ def json_encoder_default(obj: Any) -> Any:
     """
     if getattr(obj, "do_not_serialize", None):
         return None
-    if isinstance(obj, ValueDecodeFailure):
```

Comment:
Can this not happen anymore or will it be handled by the Exception check?

Reply:
Can no longer happen, I already filter it out in the subscription and interview logic